Assets/PatchKit Patcher/Scripts/AppStarter.cs

```diff
@@ -97,6 +100,10 @@ private void StartAppVersion(AppVersion? appVersion, string customArgs)
             {
                 appArgs += " " + appVersion.Value.MainExecutableArgs;
             }
+            else
+            {
+                appArgs += " " + _app.LocalMetaData.GetMainExecutableArgs();
```

Comment:
What happens here if `_app.LocalMetaData.GetMainExecutableArgs()` will be a null?
This behavior is unclear. Please do `else if(string.IsNullOrEmpty(...))` and then assign.

Reply:
I suggest introducing `mainExecutableArgs` just like in the function above.

Reply:
done